addons/website_event_sale/tests/test_frontend_buy_tickets.py

```diff
@@ -132,3 +134,91 @@ def test_buy_last_ticket(self):
     def test_pricelists_different_currencies(self):
         self.start_tour("/", 'event_sale_pricelists_different_currencies', login='admin')
     # TO DO - add public test with new address when convert to web.tour format.
+
+
+@odoo.tests.common.tagged('post_install', '-at_install')
+class TestRoutes(HttpCaseWithUserDemo, TestWebsiteEventSaleCommon, PaymentHttpCommon):
```

Comment:
Can't you share some setup with TestUi of other classes to avoid writing yet another bootstrap code ?

Reply:
Moved `event_2` and `ticket_2` didn't want to do more as it would be more specific to my use case.